src/toucan/db.clj

```diff
@@ -281,15 +313,32 @@
         (maybe-qualify model field)))
     (models/default-fields (resolve-model model))))
 
+(defn- replace-underscore [^Keyword k]
+  (let [k-str (name k)]
+    (if (string/index-of k-str \_)
+      (keyword (string/replace k-str \_ \-))
+      k)))
+
+(defn- transform-keys [f m]
+  (walk/postwalk
+    (fn [x]
+      (if (map? x)
+        (into {} (map
+                   (fn [[k v]]
+                     [(f k) v])
+                   x))
```

Comment:
Using `for` here instead of `map` + `fn` would be more readable IMO. https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#prefer-higher-level-looping-constructs